features/login.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Logging in, Logging out
+
+  @javascript
+  Scenario: I can log-in and out
+    Given I am logged in as an admin
+    And I am on the Dashboard
+    Then I should see "Howdy"
+
+    Given I am an anonymous user
```

Comment:
I think I thought our "log in as user" commands always did a log-out, but upon review, I can see anonymous is a special case, and that already does a logout, so forget I said anything! :)